packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyUtils.ts

```diff
@@ -82,7 +82,7 @@ const extractPrimitives = (
       primitiveNavigationClass,
       actualECClassName,
       parentPropertyClassName,
-      key: `${actualECClassName}__${newPropertyTraversal.join("_")}`,
+      key: `${parentPropertyClassName}_${actualECClassName}_${newPropertyTraversal.join("_")}`,
```

Comment:
Use a separator character that would not appear in ECName (https://www.itwinjs.org/bis/ec/ec-name/). I think a good candidate would be `|`.

Reply:
Changed it to `|`.